_content/lorawan/index.md

```diff
@@ -41,6 +41,10 @@ LoRaWAN operates in unlicensed radio spectrum. This means that anyone can use th
 
 LoRaWAN uses lower radio frequencies with a longer range. The fact that frequencies have a longer range also comes with more restrictions that are often country-specific. This poses a challenge for LoRaWAN, that tries to be as uniform as possible in all different regions of the world. As a result, LoRaWAN is specified for a number of bands for these regions. These bands are similar enough to support a region-agnostic protocol, but have a number of consequences for the implementation of the backend systems.
 
++ LoRaWAN has official regional specifications, called **Regional Parameters**, that you can download from the [LoRa Alliance website](https://lora-alliance.org/lorawan-for-developers).
+
++ These LoRaWAN regional specifications do not specify everything either. Indeed, they only cover a global region and do not account for specific sub-regional restrictions. For example, the LoRaWAN regional parameters for Asia only specify a common subset of channels - but there are variations between regulations in Asian countries. Furthermore, each network server operator is free to select additional parameters, such as additional emission channels. We call these parameters **Frequency Plans**. For The Things Network, they are defined in [this GitHub repository](https://github.com/TheThingsNetwork/gateway-conf).
```

Comment:
- "global region" is very confusing in this context; omit the word global
- They do account for sub-regional restrictions by specifying the common denominator